src/main.ts

```diff
@@ -5,10 +5,7 @@ import { AppService } from './app/app.service';
 
 const bootstrap = async () => {
     const port = process.env.PORT;
-    // Need to disable body parser for http-proxy to work for POSTs: https://github.com/nestjs/nest/issues/405
-    const app = await NestFactory.create(AppModule, {
-        bodyParser: false,
-    });
+    const app = await NestFactory.create(AppModule);
```

Comment:
No concerns with this, but for my own education can you just explain the change?

Reply:
Ah.  Making it consistent with https://github.com/kiva/aries-guardianship-agency/blob/main/src/main.ts

Reply:
Right on. Thanks!